symengine/printer.cpp

```diff
@@ -620,7 +623,16 @@ void StrPrinter::bvisit(const Log &x)
 
 void StrPrinter::bvisit(const Constant &x)
 {
-    str_ = x.get_name();
+    /*If the constant is E then it is to
+      be printed as exp(1), this is done
+      to maintain consistency with sympy*/
+    if(x.get_name() == "E") {
```

Comment:
Format using clang-format. See https://github.com/symengine/symengine/wiki/Development-workflow-for-SymEngine#committing-code-to-be-done-before-each-commit